app/models/StandardAuthRetrievals.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.Logger
+import uk.gov.hmrc.auth.core.AffinityGroup.Agent
+import uk.gov.hmrc.auth.core.{AffinityGroup, Enrolment, EnrolmentIdentifier}
+import uk.gov.hmrc.http.UnauthorizedException
+
+case class StandardAuthRetrievals(enrolments: Set[Enrolment],
+                                  affinityGroup: Option[AffinityGroup],
+                                  delegationModel: Option[DelegationModel]) {
+
+  private val agentKey = "HMRC-AGENT-AGENT"
```

Comment:
AMBER - All these val's are only used once, don't know whether we need to explicitly name them.

Reply:
Rejected - leaving with private val references